src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    Circle(Color color, double radius) {

```

Comment:
The constructor for Circle should be public to ensure it can be instantiated from other classes if needed.